git-ftp

```diff
@@ -383,7 +384,12 @@ get_config() {
 }
 
 set_deployed_sha1_file() {
-	DEPLOYED_SHA1_FILE="$(get_config deployedsha1file "$DEPLOYED_SHA1_FILE")"
+	DEPLOYED_SHA1_FILE_CONFIG="$(get_config deployedsha1file)"
+	if [ -z "$DEPLOYED_SHA1_FILE_CONFIG" ]; then
+	    DEPLOYED_SHA1_FILE="$(get_config deployedsha1file "$DEPLOYED_SHA1_FILE")" #If none in config, rerun with harcoded value
+    else
+        DEPLOYED_SHA1_FILE=$DEPLOYED_SHA1_FILE_CONFIG
+	fi
```

Comment:
I don't understand this change. The old line should have done exactly the same. If the config variable is set then `$(get_config deployedsha1file "$DEPLOYED_SHA1_FILE")` will return the config option. If it's not set then it will return the original value of `$DEPLOYED_SHA1_FILE`.